streams-contrib/streams-provider-twitter/src/main/java/org/apache/streams/twitter/provider/TwitterFollowingProvider.java

```diff
@@ -129,7 +134,7 @@ public StreamsResultSet readCurrent() {
 
     @Override
     public void prepare(Object o) {
```

Comment:
Not really clear what this method does, but it doesn't seem to do anything with its Object o parameter any more. Should this be removed?